src/algorithms/calorimetry/CalorimeterHitReco.cc

```diff
@@ -276,6 +276,7 @@ void CalorimeterHitReco::process(
             sid,
             lid,
             local_position);
+        recohits->setRawHit(rh);
```

Comment:
For the final version of this PR, we need to update those to support older EDM's.
```suggestion
#if EDM4EIC_VERSION_MAJOR >= 7
        recohits->setRawHit(rh);
#endif
```

and
```c++
#include <edm4eic/EDM4eicVersion.h>
```